core-bundle/src/Cron/ScopedCronJobInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Cron;
+
+interface ScopedCronJobInterface
```

Comment:
This should be named `ScopeAwareCronJobInterface` to be consistent with lots of Symfony libraries. Everything that accepts a setter is "aware" of it :)